featurebyte/models/online_store_compute_query.py

```diff
@@ -38,6 +39,10 @@ class OnlineStoreComputeQueryModel(FeatureByteCatalogBaseDocumentModel):
     serving_names: List[StrictStr]
     feature_store_id: Optional[PydanticObjectId] = Field(default=None)
 
+    @property
+    def warehouse_tables(self) -> list[TableDetails]:
+        return [TableDetails(table_name=self.tile_id), TableDetails(table_name=self.table_name)]
```

Comment:
Multiple `OnlineStoreComputeQueryModel` can reference the same `tile_id`. Would that be an issue?

Reply:
It should be fine as we will collect set of tables before doing actual warehouse operations.